app/assets/stylesheets/kudos/components/_kudo.scss

```diff
@@ -21,6 +21,32 @@
   top: 1em;
 }
 
+.kudo__update {
+  position: absolute;
+  right: 1em;
+  text-align: right;
+  top: 4.25rem;
+}
+
+.edit-button {
+  @extend .btn;
+  height: 25px;
+  padding: 0;
+  text-align: center;
+  width: 50px;
+}
+
+.save-button {
```

Comment:
[BEM-esque](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) naming can help make things clear here (and avoid the extra `@extend .btn`). I think the edit and save buttons both have the class `.edit-button` but then `.save-button` is applied in certain cases. Since `save-button` is a modification of `edit-button`, can it be named with a double dash? (e.g. `kudo__edit-btn` and `kudo__edit-btn--save`?)